index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div id="main">
```

Comment:
Kindly consider using [Semantic HTML](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwipz6_upo7-AhUiQvEDHf7FDLkQFnoECC8QAQ&url=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FGlossary%2FSemantics&usg=AOvVaw1B6qsXIooodDa3t1Uxayp1). Using semantic tags such as  `main` instead of `div` makes your code easier to understand, making the source code more readable for other developers.  Semantic HTML tags help the search engines and other user devices to determine the importance and context of web pages.😊